src/libs/actions/IOU.ts

```diff
@@ -392,6 +393,10 @@ function setMoneyRequestParticipants_temporaryForRefactor(transactionID: string,
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {participants});
 }
 
+function setSplitPayer(transactionID: string, payerAccountID: number) {
```

Comment:
What happens if we change the payer more than once? Will their accountID be pushed into the array?

Reply:
@youssef-lr It will contain only the last accountID.